homeassistant/components/tibber/__init__.py

```diff
@@ -20,33 +17,13 @@
     "sensor",
 ]
 
-CONFIG_SCHEMA = vol.Schema(
-    vol.All(
-        cv.deprecated(DOMAIN),
-        {DOMAIN: vol.Schema({vol.Required(CONF_ACCESS_TOKEN): cv.string})},
-    ),
-    extra=vol.ALLOW_EXTRA,
-)
-
 _LOGGER = logging.getLogger(__name__)
 
 
 async def async_setup(hass, config):
     """Set up the Tibber component."""
 
     hass.data[DATA_HASS_CONFIG] = config
-
```

Comment:
`hass.data[DATA_HASS_CONFIG] = config
`
Is used to configure the notification platform. Is this the best way to do it?

Reply:
Yes/no. We don't have a good way to handle notifications at this point.

Reply:
Thanks